apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -224,13 +227,14 @@ var TopInstructions = React.createClass({
       this.props.isEmbedView && styles.embedView,
     ];
     const ttsUrl = this.props.ttsMarkdownInstructionsUrl;
-
+    const videoData = this.props.levelVideos[0];
```

Comment:
Do we only ever have 1 video?

Also, levelVideos is not marked as required, but if undefined I think we'll hit errors (i.e. line 255)